contracts/SlashIndicator.sol

```diff
@@ -72,6 +85,25 @@ contract SlashIndicator is ISlashIndicator,System,IParamSubscriber, IApplication
     bytes voteAddr;
   }
 
+  // BC-fusion
+  struct BscHeader {
```

Comment:
I dont want SlashIndicator.sol to have such complicated structure.  We can refer to `TendermintLightClient.sol`, and let the pre-compile contract to do the complicated job while smart contract only do the simple job.

Reply:
Done